File: Iniciante/1018.js
let valor = Number(lines.shift());
console.log(valor);

let notas = [100, 50, 20, 10, 5, 2, 1];
let quantidades = [0, 0, 0, 0, 0, 0, 0];

for (let i = 0; i < notas.length; i++) {
  quantidade = parseInt(valor/notas[i]);
  valor %= notas[i];
  quantidades[i] = quantidade;
}

for (let i = 0; i < notas.length; i++) {
  console.log(`${quantidades[i]} nota(s) de R$ ${notas[i]},00`);
}

File: Iniciante/1018.py
valor = int(input());
print(valor);

notas = [100, 50, 20, 10, 5, 2, 1];
quantidades = [0, 0, 0, 0, 0, 0, 0];

for i in range(len(notas)):
    quantidade = valor//notas[i];
    valor %= notas[i];
    quantidades[i] = quantidade;

for i in range(len(notas)):
  print("{} nota(s) de R$ {},00".format(quantidades[i], notas[i]));
